textile-utils-api-v1/src/main/resources/textile-utils-api-v1.mixins.json

```diff
@@ -0,0 +1,11 @@
+{
+  "required": true,
+  "package": "net.textilemc.textile.mixin.recipe",
+  "compatibilityLevel": "JAVA_8",
+  "mixins": [
+    "RecipeDispatcherMixin"
```

Comment:
Why is this here?

Reply:
Oops... left over from copying that module. Will fix.

Reply:
About the ID, would you figure out what is needed for that?

Reply:
What do you mean about id? if you mean id-256, that is what vanilla uses in the blockItem registry. See the bottom of the static constructor for Block.class

Reply:
He's talking about, that we have to hardcode an ID for our block and that would conflict with any other mod which uses the same ID. We need to use a registry sync so that isn't a problem.

Reply:
I need to look more into that. 
But it would basically involve registering without a raw id (by registering them to the next available id), and on world load, remap the registry to match a registries.dat file (which stores raw id -> identifier mappings, which would then be used to remap the registry (per world)

Reply:
Maybe a new constructor which takes a String (like Identifier) would be better and then mapping to the next available digit id?

Reply:
That would cause conflicts when adding a new block

Reply:
How so?

Reply:
This would still have the 256 limit, and also if mods are loaded in a different order than before, we could have world corruption.

Reply:
A constructor which takes an Identifier (String-like), which would be mapped to the next available (not used) ID.

Reply:
also adding blocks is gonna require batching into terrain.png

Reply:
A custom block class could be used to detect if and from where we need to draw from an external png rather than from terrain.png

Reply:
@sschr15 should be adding a resource loader api, which would probably be something like `FabricLoader.getInstance().getModContainer("mymodid") + "/resources/blah.png"

Reply:
There’s a lot of features in Block that require dehardcoding 
Until there’s resource loader and texture api I’d suggest stalling

Reply:
Maybe. Speak on discord in #github for quicker responses.